src/persistence/mapiObjectStorage.ts

```diff
@@ -244,7 +244,7 @@ export class MapiObjectStorage implements IObjectStorage {
             const resource = this.paperbitsKeyToArmResource(key);
             const contentType = this.getContentTypeFromResource(resource);
             const isLocalized = localizedContentTypes.includes(contentType);
-            const item = await this.mapiClient.get<T>(`${resource}`, [await this.mapiClient.getPortalHeader("getObject")]);
+            const item = await this.apiClient.get<T>(`${resource}`, [await this.apiClient.getPortalHeader("getObject")]);
```

Comment:
We won't need these headers in Data API anymore `[await this.apiClient.getPortalHeader("getObject")]`, we will add respective logging with the new telemetry infrastructure (once you folks finish it) and designated events. Same goes to `getIsUserResourceHeader` usage.